components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -902,6 +902,130 @@ public static API getAPI(GovernanceArtifact artifact)
         return api;
     }
 
+    public static API getLightWeightAPI(GovernanceArtifact artifact)
```

Comment:
Lets add java docs to this method

Reply:
Done in  ccca5d254427d1d6d57b5c373cdf746ec51dc6e4

Reply:
Lets define artifact and lets say the possible reason for throwing the exception

Reply:
Done in dfada70927ee9a5f40be8bc04d23bf3f0bd5d280